src/assets/Pokemon.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+
+function Pokemon( { url }) {
+
+    const [pm, setPm] = useState(null);
+
+    useEffect(() => {
+
+            async function fetchData() {
+                try {
+                    const result = await axios.get(url);
+                    console.log(result.data);
+                    setPm(result.data);
+
+                } catch (e) {
+                    console.error(e);
+                }
+            }
+
+            fetchData();
+
+        }
+
+        , [url]);
+
+    return (
+        <div>
+
+            {pm &&
+            <>
+                <h1>{pm.name}</h1>
+
+                <p className="image"><img
+                    src={pm.sprites.front_default}
+                    alt={`Afbeelding`}
```

Comment:
Je hebt hier zowel `{}` als backticks gebruikt. Gezien je een woord zonder variabelen gebruikt, is `alt="Afbeelding"` voldoende!